content/docs/04.5-devx/09-manage-dev-engine/30-sso.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Manage Single Sign-On (SSO)"
+metaTitle: "Manage Single Sign-On (SSO)"
+metaDescription: "Learn how to configure SSO for Palette Dev Engine."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Overview
+
+Palette supports the ability to use Single Sign-On (SSO) and third-party Social Sign-In Providers such as Google and GitHub. You can choose to enable or disable this feature. Use the following steps to either enable or disable the feature. 
+
+# Enable SSO
+
+To enable SSO with third-party Social Sign-In Providers use the following steps.
+
+<br />
+
+
+<InfoBox>
+
+To learn more about the Sign-In Flow, refer to the [User Authentication](/user-management/user-authentication#signinflow) documentation.
+
+</InfoBox>
+
+
+## Prerequisites
+
+* Palette Tenant Administrator access.
+
+
+## Enable SSO
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a Tenant Admin.
```

Comment:
I've seen "Tenant Admin" and "tenant admin" in our docs. I'll add it to the Agenda to discuss.